src/ui/mainwindow.cpp

```diff
@@ -26,6 +28,9 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent),
                                           m_is_replaying(false) {
     m_ui->setupUi(this);
     initialize_components();
+
+    Logger::init_logging()->set_log_filepath(LOG_FILEPATH);
+    Logger::init_logging()->log(LogLevel::INFO, LogOutput::FILE, "Logger initialized and filepath set");

```

Comment:
Let's remove this line. It does not really provide any useful information. Ideally, you don't want to log everything since it can cause the log file to be spammed.